roles/artifacts/tasks/ansible_logs.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Generate list of ansible logs to collect in home directory
```

Comment:
isn't this a repetition of https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci/playbooks/collect-logs.yml#L72-L84?

Note that I'm all for having that in the artifacts instead of zuul - but we may want to clean that other play?